Cargo.toml

```
@@ -28,10 +28,12 @@ containerd-shim-wasm-test-modules = { path = "crates/containerd-shim-wasm-test-m
 crossbeam = { version = "0.8.2", default-features = false }
 env_logger = "0.10"
 libc = "0.2.149"
-libcontainer = { version = "0.2", default-features = false }
+# Move back to a published version of libcontainer when a new release is available
+libcontainer = { git = "https://github.com/containers/youki.git", rev = "fd25afb76a66b9e1cd7c4a37ae76bf46fc3f5542", default-features = false }
```

We have released v0.3.0, including your patch
```suggestion
libcontainer = { version = "0.3", default-features = false }
```